app/models/incomeSourceDetails/viewmodels/CheckBusinessDetailsViewModel.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails.viewmodels
+
+import java.time.LocalDate
+case class CheckBusinessDetailsViewModel(businessName: Option[String], businessStartDate: Option[LocalDate],
```

Comment:
looks like quite a few fields are missing:
ie.
businessDetails.accountingPeriodEndDate
please cross check with required fields for API which consume these in the spec
--

Reply:
Ok will do